src/transforms/compression.js

```diff
@@ -1,4 +1,7 @@
-var [readVarInt, writeVarInt, sizeOfVarInt] = require("../datatypes/utils").varint;
+var ProtoDef = require("protodef").ProtoDef;
+var proto = new ProtoDef();
+
+var [readVarInt, writeVarInt, sizeOfVarInt] = proto.types.varint;
```

Comment:
ah yeah I forgot about this. datatypes should be exposed by protodef directly hmm...